api/v1/client.go

```diff
@@ -1348,20 +1348,21 @@ func (c *storageClient) Validator(ctx context.Context, r *http.Request) (*Valida
 				%s.accounts.escrow_balance_active AS escrow,
 				%s.commissions.schedule AS commissions_schedule,
 				case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND voting_power > 0) then true else false end as active,
-				case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND %s.nodes.roles like 'validator') then true else false end as status,
+				case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND %s.nodes.roles like '%%validator%%') then true else false end as status,
```

Comment:
```suggestion
				CASE WHEN EXISTS(SELECT null FROM %s.nodes WHERE %s.entities.id = %s.nodes.entity_id AND %s.nodes.roles LIKE '%%validator%%') THEN true ELSE false END AS status,
```
Style-wise, we should be capitalizing all SQL keywords for readability. Sorry for not pointing this out earlier in the previous review.

Reply:
Also, why do you need `%%validator%%`? Just `%validator%` should work to get anything with `validator` in any position within the string.

Reply:
Ah, needed to escape the Golang parsing.